bot/exts/moderation/verification.py

```diff
@@ -355,6 +356,28 @@ async def _send_requests(self, members: t.Collection[discord.Member], request: R
 
         return n_success
 
+    async def _add_kick_note(self, member: discord.Member) -> None:
+        """
+        Post a note regarding `member` being kicked to site.
+
+        Allows keeping track of kicked members for auditing purposes.
+        """
+        payload = {
+            "active": False,
+            "actor": self.bot.user.id,  # Bot actions this autonomously
+            "expires_at": None,
+            "hidden": True,
+            "reason": "Verification kick",
+            "type": "note",
+            "user": member.id,
+        }
+
+        log.trace(f"Posting kick note: {payload!r}")
```

Comment:
Is logging the whole payload useful here? The only information that really changes is the user id

Reply:
Could just log the member id if you think it's too noisy. I wanted to log that the request was made and figured I'd also log the payload given that trace logs are usually off in production anyway, but you're right that it's perhaps redundant.

Reply:
I don't think it's too noisy (unless your terminal line wraps or something like that), but logging the whole payload does push out the user id a bit further, which I think is the thing you'd usually be looking at. Since it's a Member object we could for example log `f"Posting kick note for user {member} ({member.id})"` to expand on the id

Reply:
It comes with the slight risk that someone changes the payload but forgets to update the log, causing it to lie. But you're right that it obfuscates the important information. I'll adjust it.

Reply:
Looks better on 54fb163:

![image](https://user-images.githubusercontent.com/44734341/96313092-5fa1f180-100d-11eb-9220-a4b81a5fb545.png)